Gems/Terrain/Code/Source/EditorComponents/EditorTerrainPhysicsColliderComponent.cpp

```diff
@@ -79,8 +128,9 @@ namespace Terrain
 
     AZ::u32 EditorTerrainPhysicsColliderComponent::ConfigurationChanged()
     {
+        m_component.UpdateConfiguration(m_configuration);
         UpdateConfigurationTagProvider();
-        return BaseClassType::ConfigurationChanged();
+        return 0;
```

Comment:
nit: There's maybe a clearer value you can return here rather than 0 like `AZ::Edit::PropertyRefreshLevels::None`

Reply:
nice thanks! 
I saw the base class was returning 0, but can't find it anymore. 

`EditorWrappedComponentBase::ConfigurationChanged` does return `AZ::Edit::PropertyRefreshLevels::None` as well, so no idea where I got 0 from..